opencog/python/README.md

```diff
@@ -1,19 +1,14 @@
 Using Python with the AtomSpace
 ===============================
 
-Add these lines to the end of your ~/.bashrc file:
-```
-PYTHONPATH="$PYTHONPATH:/usr/local/share/opencog/python
-export PYTHONPATH
-```
+#### Installation and Usage
 
-Install the required dependencies:
-```
-cd ~/opencog/opencog/python
-sudo pip install -r requirements.txt --upgrade
```

Comment:
the `pip install` shoudld be called automatically from `make install`.

I have never ever (manually) done a pip install of our python code. This perhaps explains some bugs?